features/telemetry/protocols/netflow/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/parser/ParserBase.java

```diff
@@ -67,18 +80,57 @@
 
     private final AsyncDispatcher<TelemetryMessage> dispatcher;
 
+    private final EventForwarder eventForwarder;
+
+    private final Identity identity;
+
+    private long maxClockSkew = 0;
+
+    private long clockSkewEventRate = 0;
+
+    private LoadingCache<InetAddress, Optional<Instant>> eventCache;
+
     public ParserBase(final Protocol protocol,
                       final String name,
-                      final AsyncDispatcher<TelemetryMessage> dispatcher) {
+                      final AsyncDispatcher<TelemetryMessage> dispatcher,
+                      final EventForwarder eventForwarder,
+                      final Identity identity) {
         this.protocol = Objects.requireNonNull(protocol);
         this.name = Objects.requireNonNull(name);
         this.dispatcher = Objects.requireNonNull(dispatcher);
+        this.eventForwarder = Objects.requireNonNull(eventForwarder);
+        this.identity = Objects.requireNonNull(identity);
+
+        setClockSkewEventRate(3600);
```

Comment:
Can we make this configurable, similar to the `maxClockSkew`?

Reply:
This is configurable by parameter. This is just the default value (1 hour).

Reply:
Right.